pkg/virt-controller/watch/snapshot/snapshot.go

```diff
@@ -464,8 +472,16 @@ func (ctrl *VMSnapshotController) createContent(vmSnapshot *snapshotv1.VirtualMa
 		vb := snapshotv1.VolumeBackup{
 			VolumeName: volumeName,
 			PersistentVolumeClaim: snapshotv1.PersistentVolumeClaim{
-				ObjectMeta: *pvc.ObjectMeta.DeepCopy(),
-				Spec:       *pvc.Spec.DeepCopy(),
+				ObjectMeta: metav1.ObjectMeta{
```

Comment:
Since this appears to be done in a couple places, maybe it makes sense to have a helper function for creating the appropriate ObjectMeta (stripped on managedFields I believe)